policyengine_us/variables/gov/states/nh/tax/income/nh_taxable_income.py

```diff
@@ -10,3 +10,4 @@ class nh_taxable_income(Variable):
     reference = "https://www.gencourt.state.nh.us/rsa/html/V/77/77-4.htm "
     defined_for = StateCode.NH
     adds = ["dividend_income", "interest_income"]
```

Comment:
Noting here because it surprised me by deviating from other states.
```suggestion

    # New Hampshire allows for negative taxable income.
    # It limits tax to nonnegative values in the tax computation instead.
    adds = ["dividend_income", "interest_income"]
```